activerecord/lib/active_record/railties/databases.rake

```diff
@@ -22,7 +22,7 @@ db_namespace = namespace :db do
     end
   end
 
-  desc 'Creates the database from DATABASE_URL or config/database.yml for the current RAILS_ENV (use db:create:all to create all databases in the config). Without RAILS_ENV, it defaults to creating the development and test databases.'
+  desc 'Creates the database from DATABASE_URL or config/database.yml for the current RAILS_ENV (use db:create:all to create all databases in the config). Without RAILS_ENV or when RAILS_ENV is development, it defaults to creating the development and test databases.'
   task :create => [:load_config] do
     ActiveRecord::Tasks::DatabaseTasks.create_current
```

Comment:
One alternative is pass an array of environments here and if people want to have only one database created they can override this task to pass only the current env.

Reply:
@rafaelfranca I'm thinking about the best way to get the behavior that respects `RAILS_ENV=` explicitly set on the command line. I think all of us who prefer this behavior would want it everywhere, not only for `db:create`, so it would be necessary to send a PR to Rails to change all of these tasks to pass an array of environments, and then to override all of them in our application. Would that be it?